wiki/Client/Release_stream/Lazer/en.md

```diff
@@ -28,6 +28,8 @@ tags:
 
 "Lazer" is the code name for the next major update to osu!. While the other release streams are feature-locked, lazer will continue to receive updates and eventually become the new "stable" client, with other versions renamed accordingly.
 
+Being the next major version, lazer introduces many new game features and aspects, as well as changes in existing gameplay and/or editor mechanics. For a exhaustive list, see [Feature comparison](/wiki/Client/Release_stream/Lazer/Feature_comparison).
```

Comment:
i think a hatnote would suffice

Reply:
I disagree, because the article is impossible to find if you are not looking for it specifically. this should boost its discoverability at least a little

Reply:
aren't hatnotes supposed to make it *more* visible?

```markdown
*For an exhaustive list of new features and other differences compared to osu!(stable), see: [osu!(lazer) feature comparison](/wiki/Client/Release_stream/Lazer/Feature_comparison).*
```

Reply:
oh, I mistook it for a footnote. will fix